applications/audio/broadcast_configuration_tool/prj.conf

```diff
@@ -27,7 +27,7 @@ CONFIG_LOG=y
 CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
 CONFIG_LOG_TAG_MAX_LEN=2
 CONFIG_LOG_TAG_DEFAULT="--"
-CONFIG_LOG_BACKEND_UART=n
+CONFIG_LOG_BACKEND_RTT=y
```

Comment:
Discuss. It is simpler for most I believe to use UART and we don't need the high bandwidth/low latency here.

Reply:
I believe this was done to not get "double up" of the prints when using the shell. The log will still come out through the shell, so it might be a bug in Zephyr